lib/Base/TensorSerialization.cpp

```diff
@@ -15,12 +15,13 @@
  */
 
 #include "glow/Base/TensorSerialization.h"
-
+#include "glow/Base/Image.h"
+#include "llvm/Support/CommandLine.h"
```

Comment:
nit: Style is to put a line break to separate glow, llvm, and system includes (otherwise clang-format will alphabetize them and may put them in an order we don't want).